src/docs/admin/references/NodeSourcePolicyReference.adoc

```diff
@@ -59,12 +59,12 @@ removed by user request. To use this policy, you have to precise the following p
 **   +ALL+ - Everybody
 
 
-===== Time Slot Policy
+==== Time Slot Policy
 
 *Time slot policy* is aimed to acquire nodes for particular time with an
 ability to do it periodically. To use this policy, you have to precise the following parameters:
 
--   +nodeUsers+ - utilization permission defined who can get nodes for
+-   *nodeUsers* - utilization permission defined who can get nodes for
```

Comment:
we should do a replace all from "utilization permission" to "Utilization permission" (capital U)